python/modules/task.py

```diff
@@ -222,27 +222,32 @@ class TASK_MERGE_SUBDIR_CSV(TASK):
     def __init__(self, parameters, updates, args = {}):
         TASK.__init__(self, parameters, updates, args)
 
+    def _merge_csv(self, csv_name, main_subdir_list, dict_setts):

```

Comment:
This function should be moved to module csv_managemement. Function merge_csv in external module should not use dict_setts. 
```
def _merge_csv(csv_name, main_subdir_list, delimiter, column_name, output_path):
```